libp2p/connmanager.nim

```diff
@@ -212,7 +213,7 @@ proc closeMuxer(muxer: Muxer) {.async.} =
     try:
       await muxer.handler # TODO noraises?
     except CatchableError as exc:
-      trace "Exception in close muxer handler", exc = exc.msg
+      trace "Exception in close muxer handler", errMsg = exc.msg
```

Comment:
Are we sure we want to use the name `errMsg` for logs that are `trace` or `debug` level? Seems kinda weird to me. Also would make filtering logs with `grep` using `err` show not just actual errors but more like low level warning messages about poor connections with peers.

Thanks for the fix, but seems to me like using the keyword `err` in this kinda misses the point of having log levels.

Reply:
I agree, how about `description`?

Reply:
Sounds good to me.

Reply:
Fair point, didn't think about the `| grep ERR` possibility. Fine by me, will change it.